src/main/kotlin/org/jitsi/rtp/rtp/header_extensions/SdesHeaderExtension.kt

```diff
@@ -40,17 +40,14 @@ class SdesHeaderExtension {
 
         private fun getTextValue(buf: ByteArray, offset: Int): String {
             val dataLength = getDataLengthBytes(buf, offset)
-            val copy = BufferPool.getArray(dataLength)
-            System.arraycopy(buf, offset + SdesHeaderExtension.DATA_OFFSET, copy, 0, dataLength)
-            return String(copy, 0, dataLength, StandardCharsets.US_ASCII)
+            return String(buf, offset + SdesHeaderExtension.DATA_OFFSET, dataLength, StandardCharsets.UTF_8)
         }
 
         private fun setTextValue(buf: ByteArray, offset: Int, sdesValue: String) {
             val dataLength = getDataLengthBytes(buf, offset)
             assert(dataLength == sdesValue.length) { "buffer size doesn't match SDES value length" }
```

Comment:
As discussed since SDP specs RFC 5888 and 4566 limit this to ASCII only (in fact even only a subset of it) and Jicofo is the offerer and thus sets the value we can keep our code a lot simpler by only supporting ACSII for now.